README.rst

```diff
@@ -43,6 +43,24 @@ Credential
 Support `AWS CLI credentials
 configuration <http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html>`__.
 
+
```

Comment:
done, but i would put sth like this above the credentials, because this is arguably the easiest way to fast-forward pull data in a structured form and doesnt require global credentials.